keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -362,8 +385,8 @@ proposal will be implemented, this is the place to discuss them.
 -->
 ### Exporting Services
 
-Services will not be visible to other clusters in the clusterset by default.
-They must be explicitly marked for export by the user. This allows users to
+Services will not be visible to other clusters by default.
+They must be explicitly marked for export. This allows users to
```

Comment:
I think being marked for export by the user is an important part of the original that should be retained.